phpBB/styles/prosilver/theme/images/site_logo_blue.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 291 96">
+    <g fill="#339DD5" fill-rule="evenodd">
```

Comment:
can I ask where you got this svg? the color does not seem to be correct? https://www.phpbb.com/about/logos/

Also, this should probably be moved to styles/alll/img. As that is where it will live in the new theme and prevent issues from breaking linkage in future versions.

Reply:
Could not find the color documented anywhere (nor did I fell like going on a hunt the likes of which could make for another National treasure movie) so I stole it from the top of our blue gradient logo.

Reply:
styles/all/img does not exist..in this branch. This is just a blue version of the existing site_logo already in the prosilver folder, which in both this branch and the master branch, is in prosilver/theme/images